src/main/java/io/vertx/rabbitmq/impl/RabbitMQClientImpl.java

```diff
@@ -46,9 +40,46 @@ public RabbitMQClientImpl(Vertx vertx, JsonObject config) {
   }
 
   @Override
+  public boolean isConnected() {
+    boolean connected = false;
+    if (connection != null) {
+      if (connection.isOpen()) {
+        connected = true;
+      }
+    }
+    return connected;
+  }
+
+  @Override
+  public boolean isOpenChannel() {
+      return channel.isOpen();
+  }
+
+  @Override
+  public void basicAck(long deliveryTag, boolean multiple, Handler<AsyncResult<JsonObject>> resultHandler) {
+    forChannel(resultHandler, (channel) -> {
+      channel.basicAck(deliveryTag, multiple);
+      return null;
+    });
+  }
+
+  @Override
+  public void basicNack(long deliveryTag, boolean multiple, boolean requeue, Handler<AsyncResult<JsonObject>> resultHandler) {
+    forChannel(resultHandler, (channel) -> {
+      channel.basicNack(deliveryTag, multiple, requeue);
+      return null;
+    });
+  }
+
+  @Override
   public void basicConsume(String queue, String address, Handler<AsyncResult<Void>> resultHandler) {
+    basicConsume(queue, address, true, resultHandler);
+  }
+
+  @Override
+  public void basicConsume(String queue, String address, boolean autoAck, Handler<AsyncResult<Void>> resultHandler) {
     forChannel(resultHandler, channel -> {
-      channel.basicConsume(queue, new ConsumerHandler(vertx, channel, includeProperties, ar -> {
+      channel.basicConsume(queue, new ConsumerHandler(vertx, channel, includeProperties, autoAck, ar -> {
```

Comment:
Hm shouldn't we call basicConsume(queue, true, new ConsumerHandler(....)); in case we want to have autoAck? This is more performant since the rabbitmq broker doesn't have to wait for any ACKs received from this consumer (helps with throughput). In that case the ConsumerHandler obviously has to refrain from acking the message.